specs/v1.yaml

```diff
@@ -61,12 +61,15 @@ paths:
           $ref: "#/components/responses/InternalServerError"
   /v1/nonprofits:
     post:
+      deprecated: true
       summary: Create nonprofit
       description: |-
```

Comment:
Can we add a note to the description regarding the deprecation notice and refer users to just use the GET api instead?

Reply:
done